kafka_broker.yml

```diff
@@ -0,0 +1,85 @@
+---
+- name: Kafka Broker Status Finding
+  hosts: kafka_broker
+  gather_facts: false
+  tags: kafka_broker
+  environment: "{{ proxy_env }}"
+  tasks:
+    - import_role:
+        name: confluent.variables
+
+    - name: Check if Kafka Service Running
+      shell: "systemctl show -p SubState {{kafka_broker_service_name}}"
+      changed_when: false
+      check_mode: false
+      register: substate
+
+    - set_fact:
+        install_pattern: "{{ 'parallel' if substate.stdout != 'SubState=running' or kafka_broker_reconfiguration_pattern == 'parallel' else 'serial' }}"
+
+    - name: "Group Hosts by Installation Pattern: Parallel or Serial"
+      group_by:
+        key: kafka_broker_{{install_pattern}}
+
+- name: Kafka Broker Parallel Provisioning
+  hosts: kafka_broker_parallel
+  gather_facts: false
+  tags: kafka_broker
+  environment: "{{ proxy_env }}"
+  tasks:
+    - import_role:
+        name: confluent.kafka_broker
+
+- name: Kafka Broker Serial Ordering
+  hosts: kafka_broker_serial
+  gather_facts: false
+  tags: kafka_broker
+  environment: "{{ proxy_env }}"
+  vars_files:
+    - roles/confluent.kafka_broker/defaults/main.yml
+  tasks:
+    - import_role:
+        name: confluent.variables
+
+    - name: Get Controller Broker ID
```

Comment:
Should we look at getting the controller ID via the REST endpoint?  With zookeeper going away sooner then later, would make sense to do this work now. https://confluentinc.atlassian.net/wiki/spaces/KAFKA/pages/929205481/Rest+API+in+the+Broker+to+expose+its+cluster+id

Reply:
Understood.  Something else to add to our list in the future. :)

Reply:
According to that blog, can also check the `metadata.properties` file, I'll investigate

Reply:
lookey lookey:
```
[root@kafka-broker1 data]# cat /var/lib/kafka/data/meta.properties
#
#Tue Jan 05 17:39:43 UTC 2021
cluster.id=qIDfNCw_Sr-fIAT37l9S_A
version=0
broker.id=1
```

Reply:
validated with 2 datadirs there will be a meta.properties in both, containing the same clusterid
```
[root@kafka-broker1 data1]# grep cluster /var/lib/kafka/data1/meta.properties
cluster.id=-5ZcM36rTLWmsy3fHxRxYA
[root@kafka-broker1 data1]# grep cluster /var/lib/kafka/data2/meta.properties
cluster.id=-5ZcM36rTLWmsy3fHxRxYA
```

Reply:
Wow I just realized somehow we got from controller id to cluster id! There is clearly an easy way to get the clusterid.... but thats not what this task is populating! I'm going to leave a comment to the KIP saying how to update when ZK is removed

Reply:
This is really good to know, either way!  Will mark this as resolved, and we can address in the future.  Still a really good tip about the cluster ID though.